### PR TITLE
creating new rails application

### PR DESCRIPTION
**Pull request by Sspadea**
## Summary

- Creating a new wbooks-api rails application

## Trello Card

https://trello.com/c/CPXTvrLv/13-1-crear-aplicaci%C3%B3n-rails-wbooksapi

## Screenshot

Captura de Pantalla 2021-06-03 a la(s) 16 22 34

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
+end
```
Comment (redwarewolf) [inline app/mailers/application_mailer.rb:4]: Remove everything we aren't going to use

**Comment by redwarewolf** [comment]
@blacksam07 is it possible this PR has more files than it needs?

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem 'pg'
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
```
Comment (blacksam07) [inline config/database.yml:20]: what is the username and password for development?

**Review by blacksam07 — APPROVED** [review]

File: config/database.yml (hunk 2 above)
Comment (Sspadea) [reply config/database.yml:20]: The username is wbooks-api and the password is the same, but it is commented, that should be uncommented, right?

Comment (blacksam07) [reply config/database.yml:20]: yeah

**Pull request merged**